love/components/questions-form.tsx

```diff
@@ -151,9 +161,80 @@ const QuestionRow = (props: { row: rowFor<'love_questions'>; user: User }) => {
             setForm({ ...form, integer: Number(e.target.value) })
           }
           value={form.integer ?? undefined}
-          onBlur={() => submitAnswer()}
+          onBlur={() => submitAnswer(form)}
         />
       ) : null}
     </Col>
   )
 }
+
+export const IndividualQuestionRow = (props: {
+  row: rowFor<'love_questions'>
+  initialAnswer?: rowFor<'love_answers'>
+  user: User
+  onCancel: () => void
+  onSubmit?: (form: loveAnswerState) => void
```

Comment:
doesn't look like anyone uses the parameter of the callback, maybe leave it out?